nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutFile.java

```diff
@@ -81,6 +83,9 @@
 
     public static final String FILE_MODIFY_DATE_ATTRIBUTE = "file.lastModifiedTime";
     public static final String FILE_MODIFY_DATE_ATTR_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
+    
+    public static final Pattern RWX_PATTERN = Pattern.compile("^([r-][w-])([x-])([r-][w-])([x-])([r-][w-])([x-])$");
+    public static final Pattern NUM_PATTERN = Pattern.compile("\\d+");
```

Comment:
Optional. We could use these two regex to validate user input when the configured value doesn't use EL. Currently, `CHANGE_PERMISSIONS` allows any non-empty value.